k2/csrc/fsa_util.cc

```diff
@@ -6,9 +6,104 @@
 
 #include "k2/csrc/fsa_util.h"
 
+#include <algorithm>
+#include <cctype>
+#include <cstdlib>
 #include <utility>
 #include <vector>
 
+#include "glog/logging.h"
+
+namespace {
+/** Convert a string to an integer.
+
+  @param [in]   s     The input string.
+  @param [out]  is    If non-null, it is set to true when the conversion
+                      is successful; false otherwise
+  @return The converted integer.
+ */
+int32_t StringToInt(const std::string &s, bool *is_ok = nullptr) {
```

Comment:
I was just thinking that maybe we sould not put those helper functions in `fsa_utils` as all functions are not related with fsa except `stringToFsa`

Reply:
I agree.

When such helper functions reach a certain number, we can place them separately.